elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/ElideHjsonUtil.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers;
+
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.hjson.JsonValue;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Util class for HJSON parsing
+ */
+@Slf4j
+public class ElideHjsonUtil {
```

Comment:
Thoughts on changing this to just `util.java` so that we can expand it for including any other methods to be used in Spring and Standalone. I think the HTTP reader and File Reader will be needed in Spring and Standalone as well.